geode-core/src/main/java/org/apache/geode/cache/asyncqueue/internal/ParallelAsyncEventQueueImpl.java

```diff
@@ -110,6 +110,7 @@ private void start(boolean cleanQueues) {
       logger.info("Started  {}", this);
 
       enqueueTempEvents();
+      startTime = System.currentTimeMillis();
```

Comment:
startTime should be set after the logger.info

Reply:
I would say before the logger.info.